app/javascript/components/public_health/CustomExport.js

```diff
@@ -214,46 +214,90 @@ class CustomExport extends React.Component {
+                            <small>
+                              <b className="mr-2">{filter.filterOption?.title}:</b>
+                              {['search', 'select', 'number'].includes(filter.filterOption?.type) && (
+                                <span>{filter.value === '' ? '<blank>' : filter.value}</span>
+                              )}
+                              {filter.filterOption?.type === 'boolean' && <span>{filter.value ? 'True' : 'False'}</span>}
+                              {filter.filterOption?.type === 'date' && (
+                                <span>
+                                  {filter.dateOption === ''
+                                    ? '<blank>'
+                                    : `${filter.dateOption} ${
+                                        filter.dateOption === 'within'
+                                          ? moment(filter.value?.start).format('MM/DD/YYYY') + ' and ' + moment(filter.value?.end).format('MM/DD/YYYY')
+                                          : moment(filter.value).format('MM/DD/YYYY')
+                                      }`}
+                                </span>
+                              )}
+                              {filter.filterOption?.type === 'relative' && (
+                                <span>
+                                  {filter.relativeOption === 'custom'
+                                    ? `in the ${filter.value?.when} ${filter.value?.number} ${filter.value?.unit}`
+                                    : filter.relativeOption}
+                                </span>
+                              )}
+                              {filter.filterOption?.type === 'multi' && (
+                                <div style={{ display: 'inline-grid' }}>
+                                  {filter.value?.map((f, i) => {
+                                    return (
+                                      <p key={`filter-${index}-${i}`} className="mb-0">
+                                        <b className="mr-2">
+                                          {filter.filterOption?.fields
+                                            ?.find(fields => {
+                                              return fields.name === f.name;
+                                            })
+                                            ?.title?.replace(/\b\w/g, l => l.toUpperCase())}
+                                        </b>
+                                        {typeof f.value === 'string'
+                                          ? f.value === ''
+                                            ? '<blank>'
+                                            : f.value
+                                          : f.value?.when === ''
+                                          ? '<blank>'
+                                          : `${f.value.when}  ${moment(filter.value.date).format('MM/DD/YYYY')}`}
```

Comment:
These two lines should be indented for readability

Reply:
this is actually out of my control because it's how eslint automatically formats it

Reply:
oh ok that's... really weird